### ISSUE TITLE
Investigate Scov2 polyprotein listed as a species in Noctua

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/minerva#486]
During the QC checks for bringing Noctua up after the 2022-05-26 outage, I noticed a suspicious entry, pp1ab Scov2, in the list of species:

![image](https://user-images.githubusercontent.com/1730534/170602651-ac0f31c2-2020-4204-9dd5-3a6b46968379.png)

I thought pp1ab was a polyprotein and that's how it looks in noctua-amigo:

![image](https://user-images.githubusercontent.com/1730534/170602749-40a38983-2334-4564-87ed-972d60562a42.png)

@balhoff @tmushayahama - can you take a look to see why this entry is included as a species?  Thanks.

Also tagging @kltm

**Comment by balhoff** [comment]
@tmushayahama how is that list created? (What service does it call to get it?) 'pp1ab Scov2' does not look like a taxon at least in the latest NEO file.

**Comment by vanaukenk** [comment]
@balhoff 
@tmushayahama uses the taxon API from minerva (/taxa)

**Comment by kltm** [comment]
As a hint, noting that the /taxa API is returning:
`{ id: "http://identifiers.org/uniprot/P0DTD1", label: "pp1ab Scov2" }`

**Comment by kltm** [comment]
Noting this found in neo.obo:
```
[Term]
id: UniProtKB:P0DTD1-PRO_0000449619
name: nsp1 Scov2
synonym: "nsp1" BROAD []
synonym: "P0DTD1-PRO_0000449619" RELATED []
synonym: "protein" RELATED []
is_a: CHEBI:33695
relationship: has_gene_template PR:000050270%7CUniProtKB%3AP0DTD1-PRO_0000449635%7CPRO_0000449635
relationship: in_taxon UniProtKB:P0DTD1 ! pp1ab Scov2
property_value: https://w3id.org/biolink/vocab/category https://w3id.org/biolink/vocab/GeneProduct
property_value: https://w3id.org/biolink/vocab/category https://w3id.org/biolink/vocab/MacromolecularMachine
```
I don't believe in_taxon is supposed to work like that.

**Comment by kltm** [comment]
Origin seems to be here: https://raw.githubusercontent.com/Knowledge-Graph-Hub/kg-covid-19/master/curated/ORFs/uniprot_sars-cov-2.gpi

**Comment by kltm** [comment]
It looks like the taxon is off by one for GPI 1.2
```
UniProtKB	P0DTD1-PRO_0000449619	nsp1	Host translation inhibitor nsp1|P0DTD1(1-180)|rep/Clv:nsp1 (SARS2)|PRO_0000449619|nsp1 (SARS2)|UniProtKB:P0DTD1, 1-180|leader protein (SARS2)|UniProtKB:P0DTC1, 1-180|non-structural protein 1 (SARS2)|nsp-1|ns1|ns-1|host translation inhibitor nsp1|Severe acute respiratory syndrome (SARS) coronavirus nonstructural protein 1	protein	taxon:2697049	UniProtKB:P0DTD1	PR:000050270|UniProtKB:P0DTD1-PRO_0000449635|PRO_0000449635
```
http://geneontology.org/docs/gene-product-information-gpi-format/

**Comment by kltm** [comment]
Related to https://github.com/geneontology/go-site/issues/1431

**Comment by balhoff** [comment]
@kltm it seems like you found the problem. But in the neo.owl I downloaded yesterday I saw `in_taxon NCBITaxon:2697049`. I wonder why the discrepancy?

**Comment by kltm** [comment]
@balhoff Yeah, there's some stuff I'm not sure about here, especially as that file has not been touched in years, so I'm not sure why it's a problem now.
I'm tagging upstream contributors @cmungall and @justaddcoffee to confirm format for GPI 1.2.

**Comment by kltm** [comment]
From @cmungall , we can go ahead and manually fix this file ourselves upstream.

**Comment by kltm** [comment]
Working branch at https://github.com/Knowledge-Graph-Hub/kg-covid-19/tree/issue_geneontology_minerva-486-fix-sars-cov-2-gpi

**Comment by kltm** [comment]
If we understand this correctly, this should be fixed on next NEO release.

**Comment by kltm** [comment]
Hm. Apparently not. Still appearing on Noctua landing page.

**Comment by pgaudet** [comment]
I dont see pp1ab in the lists of organisms in Noctua
